docs/openhabian.md

```diff
@@ -191,17 +183,19 @@ These are just some first hints.
 Be sure to read up on the [Configuration](https://www.openhab.org/docs/configuration/) section of the documentation page to learn more.
 
 {: #further-config}
-### Further Configuration Steps
+### Recommended Configuration Steps
 
-openHABian is supposed to provide a ready-to-use openHAB base system. There are however a few things we can not decide for you.
+openHABian is supposed to provide a ready-to-use openHAB base system.
+There are a few things, however, we need you to decide and act on right now at the beginning:
 
+- **Delayed Rules Loading** openHAB startup times can be annoyingly long. There's an optimization available that *delays* loading the rules, *effectively speeding up* openHAB startup. We recommend to enable this now. See [menu option: 44].
```

Comment:
Nothing specific against delayed rules, but users should be aware of this.
What happens if you keep you rules open via am Samba share or fuse mount?
Maybe a small hint in the docs that this feature actually _moves_ files and rules will only be active 2 minutes after start....
Maybe it is already in the docs, I look only at the patch.

Reply:
there's no extra docs so I added a note here

Reply:
comment in the code is good, do you think we should also state the second part here in the docs?
"quickly rename rules forth and back after 2 minutes"